packages/react-fiche-service-public/src/components/Table.js

```diff
@@ -20,6 +22,10 @@ class Table extends React.PureComponent {
       child => child.name === "Rangée" && child._.type === "normal"
     );
 
+    const isHeaderCellInRow = data.$.filter(
+      child => child.name === "Colonne"
+    ).map(colonne => colonne._.type === ROW_HEADER);
+
```

Comment:
Préfixe `is` ce serait plutôt pour un boolean?

Reply:
Haha bien vu j'ai passé 5 minutes à me demander si je devais mettre `is`. En fait c'est un tableau de booléens donc je sais pas trop pour le naming. Rajouter `Array` dans le naming ?